ci/kind/test-e2e-kind.sh

```diff
@@ -44,7 +44,7 @@ function run_test {
 
   $YML_CMD --kind --encap-mode $mode $proxy | docker exec -i kind-control-plane dd of=/root/antrea.yml
   sleep 1
-  go test -v -timeout=20m github.com/vmware-tanzu/antrea/test/e2e -provider=kind
+  go test -v -timeout=26m github.com/vmware-tanzu/antrea/test/e2e -provider=kind
```

Comment:
let's go with 30 minutes so we don't run into the issue again

Reply:
Done.